package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@paypal/identity-components",
-  "version": "5.0.4",
+  "version": "5.0.4-alpha4",
```

Please back out the dist file changes and this version change as well.
```suggestion
  "version": "5.0.4",
```

---

removed